contrib/dpu_daemon/scripts/config.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash -eE
+#
+# Configuration for running script ./run.sh to build and test DPU server
+#
+
+export WORK_DIR="${WORK_DIR:-/GIT/WORK/${ARCH}}" # Base work dir
+export BUILD_DIR="${WORK_DIR}/build"
+export SCRIPT_DIR="${PWD}"
+export DOWNLOAD="${DOWNLOAD:-1}"
+#= The name of IB on the host
+export MLX_DEV_HOST="mlx5_4:1"
+
+#= Build components
+declare -a BUILD_INDEX=(UCX UCC OMPI OMB DPU)
+#= Configuration of build components
+declare -A BUILD=(
+  [dpu, ${BUILD_INDEX[0]}]=1 #UCX
+  [dpu, ${BUILD_INDEX[1]}]=1 #UCC
+  [dpu, ${BUILD_INDEX[2]}]=1 #OMPI
+  [dpu, ${BUILD_INDEX[3]}]=0 #OMB
+  [dpu, ${BUILD_INDEX[4]}]=1 #DPU
+
+  [x86, ${BUILD_INDEX[0]}]=1 #UCX
+  [x86, ${BUILD_INDEX[1]}]=1 #UCC
+  [x86, ${BUILD_INDEX[2]}]=1 #OMPI
+  [x86, ${BUILD_INDEX[3]}]=1 #OMB
+  [x86, ${BUILD_INDEX[4]}]=0 #DPU
+)
+
+#= List of packages
+declare -a PKG_INDEX=(${BUILD_INDEX[@]:0:4})
+
+#= Configurations for 'git clone'
+declare -A GIT=(
+  #UCX
+  [${PKG_INDEX[0]}, URL]='https://github.com/openucx/ucx.git'
+  [${PKG_INDEX[0]}, BRANCH]="${UCX_BRANCH:-v1.14.x}"
```

Comment:
UCX v1.15.x is available now - should we switch?

Reply:
We still have to use v1.14 with UCC branch v0.4.x